Cargo.toml

```diff
@@ -26,7 +26,7 @@ secp-recovery = ["secp256k1/recovery"]
 # Instead no-std enables additional features required for this crate to be usable without std.
 # As a result, both can be enabled without conflict.
 std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std"]
-no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc"]
+no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc", "secp256k1/alloc"]
```

Comment:
I am a little puzzled why it's needed now or how it was working without this?

Reply:
It worked because we are not creating contexts in `rust-bitcoin` itself via `Secp256k1::new`. 

However, since we are re-exporting `secp256k1`, and depending crates (such as `lightning`) do want to create contexts, it's much nicer to pull this feature in, so they don't have to declare the dependency on `secp256k1` themselves, just in order to add the feature.

Reply:
nice, thanks